src/main/java/com/github/stickerifier/stickerify/bot/Stickerify.java

```diff
@@ -32,21 +33,25 @@
  *
  * @author Roberto Cella
  */
-public class Stickerify extends TelegramBot {
+public class Stickerify {
 
 	private static final Logger LOGGER = LoggerFactory.getLogger(Stickerify.class);
 	private static final String BOT_TOKEN = System.getenv("STICKERIFY_TOKEN");
 
+	private TelegramBot bot;
```

Comment:
This should be final
```suggestion
	private final TelegramBot bot;
```